cmd/crictl/exec.go

```diff
@@ -67,6 +68,12 @@ var runtimeExecCommand = &cli.Command{
 			Aliases: []string{"i"},
 			Usage:   "Keep STDIN open",
 		},
+		&cli.StringFlag{
+			Name:    "transport",
+			Aliases: []string{"p"},
+			Usage:   "transport protocol, One of: spdy|websocket|default",
+			Value:   "default",
```

Comment:
I think we should keep `spdy` as default value and remove `default`.

Reply:
fixed